java/arcs/android/devtools/DevToolsService.kt

```diff
@@ -36,6 +39,8 @@ import kotlinx.coroutines.suspendCancellableCoroutine
 @OptIn(ExperimentalCoroutinesApi::class)
 open class DevToolsService : Service() {
 
+    val CHANNEL_ID = "DevToolsChannel"
```

Comment:
Can this be private? and it would be better as a `const val` in the companion object.